### ISSUE TITLE
SmallInteger

### ISSUE THREAD

**Issue by timoteo7** [mpociot/laravel-test-factory-helper#24]
Migration:
$table->smallInteger('small');

Factory:
'small'=> $faker->numberBetween($min = 0, $max = 32767),

**Comment by jrmadsen67** [comment]
NOTE: 
Small integer. The signed range is -32768 to 32767. The unsigned range is 0 to 65535

**Comment by timoteo7** [comment]
yes, sorry for the bad example, but can you put this too in factory helper ?

**Comment by jrmadsen67** [comment]
thanks, I understand now. I didn't know if you were pointing out a bug or
asking for a feature
